systemd/armv7hf/jessie/entry.sh

```diff
@@ -9,10 +9,17 @@ if [ "$INITSYSTEM" = "on" ]; then
 	echo -e "${GREEN}Systemd init system enabled."
 	env > /etc/docker.env
 
-	echo -e "#!/bin/bash\n exec $@" > /etc/resinApp.cmd
+	cat <<-EOF > /etc/resinApp.cmd
```

Comment:
Actually it seems unchanged from before, it just `cat`s a heredoc instead, so I guess it can stay as is